src/app/technical-record/dimensions/dimensions.component.html

```diff
@@ -1,102 +1,128 @@
 <table class="govuk-table table-details">
   <thead class="govuk-table__head">
-  <tr class="govuk-table__row">
-    <th class="govuk-table__cell govuk-!-font-weight-bold">Length (mm)</th>
-    <td id="test-length" class="govuk-table__cell">
-      {{ activeRecord.dimensions.length | DefaultNullOrEmpty }}
-    </td>
-    <th class="govuk-table__cell"></th>
-    <th class="govuk-table__cell"></th>
-    <th class="govuk-table__cell"></th>
-    <th class="govuk-table__cell"></th>
-  </tr>
+    <tr class="govuk-table__row">
+      <th class="govuk-table__cell govuk-!-font-weight-bold">Length (mm)</th>
+      <td id="test-length" class="govuk-table__cell">
+        {{ activeRecord?.dimensions.length | DefaultNullOrEmpty }}
```

Comment:
better to have as this activeRecord?.dimensions?.length. length will cause the page to crash if not available

Reply:
fixed